python/lsst/sdm/schemas/efd_latiss.yaml

```diff
@@ -0,0 +1,2764 @@
+    datatype: float
+    nullable: true
+    description: Mean relative humidity in AuxTel-ESS02.
+  - name: "at_salindex201_pressure_0_mean"
+    "@id": "#exposure_efd.at_salindex201_pressure_0_mean"
+    datatype: float
+    nullable: true
+    description: Mean atmosferic pressure item 0 in AuxTel-ESS02.
+  - name: "at_salindex201_pressure_1_mean"
+    "@id": "#exposure_efd.at_salindex201_pressure_1_mean"
+    datatype: float
+    nullable: true
+    description: Mean atmosferic pressure item 1 in AuxTel-ESS02.
+  - name: "at_salindex301_pressure_0_mean"
+    "@id": "#exposure_efd.at_salindex301_pressure_0_mean"
+    datatype: float
+    nullable: true
+    description: Mean atmosferic pressure item 0 in weather tower atmospheric pressure.
+  - name: "at_salindex301_pressure_1_mean"
+    "@id": "#exposure_efd.at_salindex301_pressure_1_mean"
+    datatype: float
+    nullable: true
+    description: Mean atmosferic pressure item 1 in weather tower atmospheric pressure.
+  - name: "at_pointing_mount_position_ra_mean"
+    "@id": "#exposure_efd.at_pointing_mount_position_ra_mean"
+    datatype: float
+    nullable: true
+    description: Mean RA calculated from the azimuthCalculatedAngle and elevationCalculatedAngle.
+  - name: "at_pointing_mount_position_ra_stddev"
+    "@id": "#exposure_efd.at_pointing_mount_position_ra_stddev"
+    datatype: float
+    nullable: true
+    description: Standard deviation of the RA calculated from the azimuthCalculatedAngle and elevationCalculatedAngle.
+  - name: "at_pointing_mount_position_dec_mean"
+    "@id": "#exposure_efd.at_pointing_mount_position_dec_mean"
+    datatype: float
+    nullable: true
+    description: Mean Dec calculated from the azimuthCalculatedAngle and elevationCalculatedAngle.
+  - name: "at_pointing_mount_position_dec_stddev"
+    "@id": "#exposure_efd.at_pointing_mount_position_dec_stddev"
+    datatype: float
+    nullable: true
+    description: Standard deviation of the Dec calculated from the azimuthCalculatedAngle and elevationCalculatedAngle.
+  - name: "at_pointing_mount_position_sky_angle_mean"
+    "@id": "#exposure_efd.at_pointing_mount_position_sky_angle_mean"
+    datatype: float
+    nullable: true
+    description: Mean c sky angle.
+  - name: "at_pointing_mount_position_sky_angle_stddev"
+    "@id": "#exposure_efd.at_pointing_mount_position_sky_angle_stddev"
+    datatype: float
+    nullable: true
+    description: Standard deviation of the calculated sky angle.
+
+- name: exposure_efd_unpivoted
+  "@id": "#exposure_efd_unpivoted"
+  description: Unpivoted EFD exposure data.
+  primaryKey:
+  - "#exposure_efd_unpivoted.exposure_id"
+  - "#exposure_efd_unpivoted.property"
+  - "#exposure_efd_unpivoted.field"
+  columns:
+  - name: exposure_id
+    "@id": "#exposure_efd_unpivoted.exposure_id"
+    datatype: long
+    nullable: false
+    autoincrement: false
+    description: Unique identifier for the exposure
```

Comment:
Missing period, if that is the style for these schemas. Please normalize across all of these schemas. (Either is fine as long as it is consistent.)

Reply:
Fixed.